docs/metrics.md

```diff
@@ -0,0 +1,113 @@
+# METRICS
+
+## Data Analysis
+
+The collected data will primarily be used to answer the following questions.
+Images are used for visualization and are not composed of actual data.
+
+### Do people use this?
+
+What is the overall engagement of Snooze Tabs? **This is the standard Daily
+Active User (DAU) and Monthly Active User (MAU) analysis.** This captures data
+from the people who have the add-on installed, regardless of whether they are
+actively interacting with it.
+
+### Immediate Questions
+
+TBD
```

Comment:
The basic question we seek to answer is: does snooze tabs provide a useful tool for its users in Test Pilot.

* Do people snooze tabs and at what rate? 
	* x = time, y = number of snoozes (vanity metric)
	* x = time, y = snoozes per user
* Do people close returned tabs without focusing them and at what rate?
	* x= time, y = snoozes focused and snoozes dismissed w/o focus
* Are the measurably different rates of returned tab engagement for snoozes of different lengths.
	* bar graph showing rate at which different snooze length tabs are focused and closed (should show custom times as well (these can be clustered into one group)
* What is the overall distribution of snooze times across all users?
	* Bar graph showing distribution of snooze times
* Do people set custom times, and at what rate, 
	* x = time, y = number of custom snooze events
* Do people re-snooze previously snoozed tabs?
	* x = time, y = snoozes of previously snoozed tabs per user